File: mvn-advent-of-code/src/test/java/advent/twenty_seventeen/Day05AMazeOfTwistyTrampolinesAllAlikeTest.java
package advent.twenty_seventeen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.*;

public class Day05AMazeOfTwistyTrampolinesAllAlikeTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day05AMazeOfTwistyTrampolinesAllAlike("/twenty_seventeen/Day05-AMazeOfTwistyTrampolinesAllAlike-input.txt");

        assertEquals(381680, dp.getPart1Answer().intValue());
        assertEquals(29717847, dp.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Integer, Integer> dp = new Day05AMazeOfTwistyTrampolinesAllAlike("/twenty_seventeen/Day05-AMazeOfTwistyTrampolinesAllAlike-example#1.txt");

        assertEquals(5, dp.getPart1Answer().intValue());
        assertEquals(10, dp.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_seventeen/Day05AMazeOfTwistyTrampolinesAllAlikeTest.scala
package advent.twenty_seventeen

import org.scalatest.{FlatSpec, Matchers}

class Day05AMazeOfTwistyTrampolinesAllAlikeTest  extends FlatSpec with Matchers {
  "2017 Day 05 - Input" should "calculate answers" in {
    val dp = new Day05AMazeOfTwistyTrampolinesAllAlike("/twenty_seventeen/Day05-AMazeOfTwistyTrampolinesAllAlike-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day05AMazeOfTwistyTrampolinesAllAlike("/twenty_seventeen/Day05-AMazeOfTwistyTrampolinesAllAlike-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
